src/pages/Homepage/homepage.jsx

```diff
@@ -179,13 +53,14 @@ return (
                     :
                     null
                     }
-                    <p className="card-content">By Megha Joshi</p>
+                    <p className="card-content">{video.creator}</p>
                     <div className="card-footer">
-                        <p className="card-content">3K views</p>
-                        <p className="card-content">3 June, 2022</p>
+                        <p className="card-content">{video.views}</p>
+                        <p className="card-content">{video.date}</p>
```

Comment:
Can use optional chaining[?.] to safely access the values inside the videos
`{video?.creator}`

Reply:
Okay